src/gui/properties/propertieswidget.h

```diff
@@ -55,15 +55,18 @@ class QAction;
 class QTimer;
 QT_END_NAMESPACE
 
-class PropertiesWidget: public QWidget, private Ui::PropertiesWidget
+class PropertiesWidget : public QWidget, private Ui::PropertiesWidget
```

Comment:
Our current uncrustify config instructs to remove the space (sp_before_class_colon), because I like it tight. We can discuss it, just open an issue.